build.gradle

```diff
@@ -22,11 +22,11 @@ allprojects {
 }
 
 ext {
-    compileSdkVersion = 25
+    compileSdkVersion = 26
     buildToolsVersion = "25.0.3"
```

Comment:
I didn't want to change the build tools since I don't know what the routines are here on that, but I can change to 26.0.1

I could also upgrade the support library to 26.0.1, but then I'd have to raise the minSDK to 14, since they removed support for <14